.vscode/cspell.json

```diff
@@ -150,12 +151,13 @@
         "LINUXOS",
         "LINUXPOOL",
         "LINUXVMIMAGE",
-        "msal",
-        "mysvc",
-        "mycluster",
         "MACOS",
         "MACPOOL",
         "MACVMIMAGE",
+        "msal",
+        "mycluster",
+        "mystorageaccount",
+        "mysvc",
```

Comment:
All of these are already in the "baseline" dictionary

Reply:
The only new words I'm adding in this PR are "mystorageaccount" and "backoff".

Do you know why if these words already exists in the "baseline" dict they're being marked as errors here https://dev.azure.com/azure-sdk/public/_build/results?buildId=4866205&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0&l=49?

Reply:
Oh Nevermind, I see that you have a separate PR already merged in main for this and https://github.com/Azure/azure-mcp/pull/202 is behind